src/Microsoft.ML.Tokenizers/PreTokenizer/PreTokenizer.cs

```diff
@@ -15,14 +13,22 @@ namespace Microsoft.ML.Tokenizers
     /// in the original string. These offsets are in the `original` referential.
     /// It also contains any `Token` associated to the current split.
     /// </summary>
-    public readonly struct Split : IEquatable<Split>
+    public struct Split : IEquatable<Split>
```

Comment:
This a public API change, addressing https://github.com/dotnet/machinelearning/issues/7009. readonly is removed.